library.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "STM32duino FreeRTOS",
+    "version": "9.0.4",
+    "keywords": "rtos, timing, thread, task, mutex, semaphore",
+    "description": "Real Time Operating System implemented for STM32",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/stm32duino/STM32FreeRTOS.git"
+    },
+    "frameworks": "arduino",
+    "platforms": "ststm32",
+    "build": {
+        "libArchive": false
```

Comment:
Do you advise to set this for each library?
I've the same PR on another library which does not specify this? is this correct?
Thanks in advance

Reply:
If a library has `weak` attributes, then is better to disable archiving.

Reply:
Ok. Thanks.